app/assets/javascripts/view-claim.js

```diff
@@ -1,111 +1,151 @@
-$(function () {
-  totalApproved()
+function totalApproved () {
+  let approvedCost = 0
+  const manuallyProcessed = 0
+  let deduction = 0
 
-  if (document.getElementById('additional-info-reject') !== null) {
-    if (document.getElementById('additional-info-reject').value === 'Other') {
-      document.getElementById('additional-info-reject-manual').style.display = 'block'
-      document.getElementById('manual-label').style.display = 'block'
-    }
+  function isValidValue (value) {
+    return !isNaN(value) && value.trim().length !== 0 && value.indexOf('e') === -1 && value.indexOf('E') === -1
   }
 
-  $('.claim-expense-status').change(function () {
-    const id = $(this).attr('data-id')
-    const value = $(this).val()
-    if (value === 'APPROVED-DIFF-AMOUNT') {
-      $(`#claim-expense-${id}-approvedcost`).removeClass('visibility-hidden').addClass('visibility-visible')
-      $(this).next('input').on('input').addClass('approved-amount')
-      $(this).parent().parent().find('td.cost').removeClass('approved-amount')
-      $('input.approved-amount').on('input', function () {
-        totalApproved()
-      })
-    } else if (value === 'APPROVED') {
-      $(`#claim-expense-${id}-approvedcost`).removeClass('visibility-visible').addClass('visibility-hidden')
-      $(this).parent().parent().find('td.cost').addClass('approved-amount')
-      $(this).next('input').on('input').removeClass('approved-amount')
-    } else {
-      $(`#claim-expense-${id}-approvedcost`).removeClass('visibility-visible').addClass('visibility-hidden')
-      $(this).parent().parent().find('td.cost').removeClass('approved-amount')
-      $(this).next('input').on('input').removeClass('approved-amount')
-    } totalApproved()
-  })
+  if ($('#unassign').length) {
```

Comment:
Not sure how I missed this, the code inside the `if` and the `else` are identical so no need for the statement?